### ISSUE TITLE
Link element is ignored when TalkBack is enabled

### ISSUE THREAD

**Issue by danylo-makarenko-bt** [mikepenz/multiplatform-markdown-renderer#487]
## Bug Description

If there are additional text elements after the link, the link is skipped when swiping left or right. While it can still be focused by clicking, it needs to be focusable during swipe navigation as well.

## Steps to Reproduce

1. Add this code
```
MarkdownText(
        """
    # Hello Markdown

    This is a simple markdown example with:

    - Bullet points
    - **Bold text**
    - *Italic text*

    [Check out this link](https://github.com/mikepenz/multiplatform-markdown-renderer)
    
    This is a simple markdown example with:

    - Bullet points
    - **Bold text**
    - *Italic text*
    
    """.trimIndent()
    )
```

2. Run on platform Android
3. See error

## Expected Behavior

All elements should be focusable and accessible by TalkBack.

## Environment

- Library Version: 0.38.1
- Platform: Android
- Device: Pixel 7 Pro
- OS Version: Android 16
- Kotlin Version: 2.1.20
- Compose Version: 2025.10.01

## Checklist

- [x] I have searched for [similar issues](https://github.com/mikepenz/multiplatform-markdown-renderer/issues)
- [x] I have checked the [sample application](https://github.com/mikepenz/multiplatform-markdown-renderer/tree/develop/sample)
- [x] I have read the [README](https://github.com/mikepenz/multiplatform-markdown-renderer/blob/develop/README.md)
- [x] I have checked the [CHANGELOG](https://github.com/mikepenz/multiplatform-markdown-renderer/releases)
- [x] I have read the [MIGRATION GUIDE](https://github.com/mikepenz/multiplatform-markdown-renderer/blob/develop/MIGRATION.md)

**Comment by mikepenz** [comment]
Thank you for the report. 

I am happy to take PRs to improve this behavior.
